CM-389: Add defaultDescription

ayumi-oxd opened the pull request:
### Jira Ticket:
CM-389

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/CM-389

### Description:
- Added new field `defaultDescription` to `activityTypes` and `facilityTypes`
- Displayed `defaultDescription` on the park and site pages if there's no `description` of `parkActivities` and `parkFacilities`

molund commented on src/staging/src/templates/park.js:
```diff
@@ -519,6 +519,7 @@ export const query = graphql`
           icon
           iconNA
           rank
+          defaultDescription
```
Does this need to be on the site template too?

ayumi-oxd replied:
I believe so. For example, this site [Wells Gray Park: Murtle Lake](https://bcparks.ca/wells-gray-park/murtle-lake/) has an empty kayak accordion.

ayumi-oxd replied:
I believed I DID but I DIDN'T 😅 Updated the site too.

molund commented on src/staging/src/components/park/parkFacility.js:
```diff
@@ -100,7 +100,11 @@ export default function ParkFacility({ data }) {
                 <div className="p-4">
-                  <HtmlContent>{facility.description}</HtmlContent>
+                  <HtmlContent>
+                    {facility.description ?
```
this should use isNullOrWhiteSpace

molund commented on src/staging/src/components/park/parkActivity.js:
```diff
@@ -99,7 +99,11 @@ export default function ParkActivity({ data }) {
                 <div className="p-4">
-                  <HtmlContent>{activity.description}</HtmlContent>
+                  <HtmlContent>
+                    {activity.description ? 
```
this should use isNullOrWhiteSpace

molund commented:
This worked for an activity (canoeing) when I tested it, but it did not work for a facility (winter camping). This is probably due to not checking isNullOrWhitespace

ayumi-oxd commented:
> This worked for an activity (canoeing) when I tested it, but it did not work for a facility (winter camping). This is probably due to not checking isNullOrWhitespace

It didn't work because I didn't set the description operator for the camping 😬
Updated `campingDetails` and added isNullOrWhitespace to check
